manifests/params.pp

```diff
@@ -10,6 +10,7 @@
       $configdir     = '/etc'
       $root_group    = 'root'
       $logrotate_bin = '/usr/sbin/logrotate'
+      $logrotate_state = '/var/lib/logrotate/logrotate.status'
```

Comment:
According to one man page I just [googled](http://www.linuxcommand.org/man_pages/logrotate8.html), the default, (if you don't specify `-s`), is `/var/lib/logrotate/status` not `/var/lib/logrotate/logrotate.status`.  I wouldn't be surprised if that varied per OS/distribution, so going with `undef` is probably safest.